### ISSUE TITLE
The format of value '.NET/BulkExecutorLibrary/1.0.1' is invalid.

### ISSUE THREAD

**Issue by JordanLeg** [Azure/azure-cosmosdb-bulkexecutor-dotnet-getting-started#28]
Hello, I founded this problem when I created two instances of the library of BulkInsert, and the expecion throws in the bulkExecutor.InitializeAsync(). My code is the following:

```

               ConnectionPolicy connectionPolicy = new ConnectionPolicy
                {
                    ConnectionMode = ConnectionMode.Direct,
                    ConnectionProtocol = Protocol.Tcp
                };

                clienteBulk = new DocumentClient(new Uri(Endpoint), Key, connectionPolicy);
                // Set retry options high during initialization (default values).
                clienteBulk.ConnectionPolicy.RetryOptions.MaxRetryWaitTimeInSeconds = 30;
                clienteBulk.ConnectionPolicy.RetryOptions.MaxRetryAttemptsOnThrottledRequests = 9;
                var coleccion = ObtenerColeccionSiExiste();
                bulkExecutor = new BulkExecutor(clienteBulk, coleccion);
                Task.Run(() => bulkExecutor.InitializeAsync()).Wait();
```

**Comment by JordanLeg** [comment]
I founded the solution only we need to call the method DocumentClient.OpenAsync before to call  
 BulkExecutor.InitializeAsync() like this

```
clienteBulk = new DocumentClient(new Uri(Endpoint), Key, connectionPolicy);
                // Set retry options high during initialization (default values).
                clienteBulk.ConnectionPolicy.RetryOptions.MaxRetryWaitTimeInSeconds = 30;
                clienteBulk.ConnectionPolicy.RetryOptions.MaxRetryAttemptsOnThrottledRequests = 9;
                var coleccion = ObtenerColeccionSiExiste();
                

> Task.Run(() => clienteBulk.OpenAsync()).Wait();

                bulkExecutor = new BulkExecutor(clienteBulk, coleccion);
               

>  Task.Run(() => bulkExecutor.InitializeAsync()).Wait();

```

**Comment by rnagpal** [comment]
This should be fixed in 1.3.0 version now.
